tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "include": ["src", "types"],
+  "compilerOptions": {
+    "module": "esnext",
+    "lib": ["dom", "esnext"],
+    "importHelpers": true,
+    "declaration": true,
+    "sourceMap": true,
+    "rootDir": "./src",
+    "strict": false,
```

Comment:
Maybe we should be as strict as possible. What do you think?

Reply:
Done in 07b9c1e5f82729e0ffa9b2c7a695498527d6013d

I had to set "strictFunctionTypes" and "noImplicitAny" to false

`strictFunctionTypes`: Conflict with `xstate` internal types
`noImplicitAny`: I'm using any on the received objects from the fullnode websocket, I could type the raw input, but we already have a `prepareTx` method to manually type the data (https://github.com/HathorNetwork/hathor-wallet-service-sync_daemon/blob/07b9c1e5f82729e0ffa9b2c7a695498527d6013d/src/utils.ts#L84)

Reply:
Can't we skip these verification specifically at these points (instead of disabling them in the whole project)?

Reply:
I agree.

Done in 3b7f2ea05186cac6680b215e34e6cf5e7ded3ae7 and 135191c30b26579d2cc043823875553b106e40e1

Reply:
That's great! Thanks!